hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/choose/algorithms/RandomPipelineChoosePolicy.java

```diff
@@ -30,6 +31,9 @@
  */
 public class RandomPipelineChoosePolicy implements PipelineChoosePolicy {
 
+  public RandomPipelineChoosePolicy(NodeManager nodeManager) {
+  }
```

Comment:
In addition to compatibility, falling back to the no-arg constructor would also let us avoid adding this.